manim/mobject/utils.py

```diff
@@ -0,0 +1,48 @@
+"""Utilities for working with mobjects."""
+
+from __future__ import annotations
+
+__all__ = [
+    "get_mobject_class",
+    "get_vectorized_mobject_class",
+]
```

Comment:
Can we make this part of Public API? That is, add it to docs and in the global import statements.

Reply:
I'm open to it; do you think users are writing renderer-independent code often enough for this to be useful for them?

I've tried to solve this in a way where creating a `VMobject` creates an `OpenGLVMobject` if the renderer is set to OpenGL -- but couldn't get that to work properly.

Reply:
> I'm open to it; do you think users are writing renderer-independent code often enough for this to be useful for them?

Hmm, I don't think it'll be helpful to users, but would be helpful for plugin developers. In which case, I would like to add it to the docs (as well as a reference in the plugin dev page), but not expose it publicly.

> I've tried to solve this in a way where creating a VMobject creates an OpenGLVMobject if the renderer is set to OpenGL -- but couldn't get that to work properly.

Looks interesting. Until that's done this would be great to have, I think.

Reply:
Added documentation (including a section in the plugin dev document) and some more doctests. Feel free to check again, otherwise I think this is ready to be merged.